shell/platform/android/android_context_gl.h

```diff
@@ -9,47 +9,49 @@
 #include "flutter/fml/memory/ref_counted.h"
 #include "flutter/fml/memory/ref_ptr.h"
 #include "flutter/shell/common/platform_view.h"
+#include "flutter/shell/platform/android/android_context.h"
 #include "flutter/shell/platform/android/android_environment_gl.h"
 #include "flutter/shell/platform/android/android_native_window.h"
 #include "third_party/skia/include/core/SkSize.h"
 
 namespace flutter {
 
-class AndroidContextGL : public fml::RefCountedThreadSafe<AndroidContextGL> {
+class AndroidContextGL : public AndroidContext {
  public:
-  bool CreateWindowSurface(fml::RefPtr<AndroidNativeWindow> window);
+  AndroidContextGL(AndroidRenderingAPI rendering_api,
+                   fml::RefPtr<AndroidEnvironmentGL> environment);
 
-  bool CreatePBufferSurface();
+  ~AndroidContextGL();
+
+  EGLSurface CreateOnscreenSurface(fml::RefPtr<AndroidNativeWindow> window);
+
+  EGLSurface CreateOffscreenSurface();
```

Comment:
Here and elsewhere, doxygen style docs please. Here I would mention this creates a 1x1 pbuffer surface used for making the offscreen context current for texture uploads.

Reply:
Done.